notebooks/QA_extraction_generation.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                                   temperature=1.5,
</pre><p>Is that the default temp value? How does the output change with this? Seems like a hyperparameter optimization task...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>